vignettes/boxr.Rmd

```diff
@@ -2,35 +2,19 @@
 title: "boxr: A high-level R interface for the box.com API"
 date: "`r Sys.Date()`"
 output: rmarkdown::html_vignette
+resource_files:
```

Comment:
What does this code do? (Just for my own personal learning)

Reply:
It's a way to tell rmarkdown directly that the it will need the `figures` directory. I don't know if it is strictly needed, as I (think I) got the problem solved another way.